impl/src/main/java/org/jboss/forge/roaster/model/impl/TypeImpl.java

```diff
@@ -272,4 +296,33 @@ private int getExtraDimensions()
       }
       return 0;
    }
+
+   public static <O extends JavaSource<O>> String ensureImports(O origin, Type<O> type)
```

Comment:
Actually, why is this method here? Shouldn't this just be a call off of JavaSource?

E.g.:

`javaClass.addImport(...)`

?